web/views/pages/index.ejs

```diff
@@ -116,10 +116,14 @@
                         <div class="layer-toggles">
                             <hr />
                             <p>
-                                <%= __('legends.carbonintensity')%> <span style="color: darkgrey">(gCO2eq/kWh) </span><small><input type="checkbox" id="checkbox-colorblind" /><label for="checkbox-colorblind"><%= __('legends.colorblindmode') %></label></small>
+                                <%= __('legends.carbonintensity')%> <span style="color: darkgrey">(gCO<sub>2</sub>eq/kWh):</span>
                                 <svg class="co2-colorbar colorbar" style="width: 100%"></svg>
                             </p>
                             <p>
+                                <input type="checkbox" id="checkbox-colorblind" />
```

Comment:
I'm not a fan of giving so much visibility to the color blind mode.
I think it should be somehow integrated to the colorbar, and not have its own line.
Depending on what you disliked in the previous version, we could discuss potential changes.
In the meantime, can you revert behavior?